test/server/overload_manager_impl_test.cc

```diff
@@ -260,14 +281,59 @@ TEST_F(OverloadManagerImplTest, CallbackOnlyFiresWhenStateChanges) {
   factory2_.monitor_->setPressure(0.42);
   timer_cb_();
   EXPECT_FALSE(is_active);
-  EXPECT_EQ(action_state, OverloadActionState::Inactive);
+  EXPECT_THAT(action_state, AllOf(Property(&OverloadActionState::isSaturated, false),
+                                  Property(&OverloadActionState::value, 0)));
   EXPECT_EQ(4, cb_count);
   EXPECT_EQ(41, pressure_gauge1.value());
   EXPECT_EQ(42, pressure_gauge2.value());
 
   manager->stop();
 }
 
+TEST_F(OverloadManagerImplTest, RangeTrigger) {
+  setDispatcherExpectation();
+
+  auto manager(createOverloadManager(getConfig()));
+  manager->start();
+  const auto& action_state =
+      manager->getThreadLocalOverloadState().getState("envoy.overload_actions.dummy_action");
+  Stats::Gauge& active_gauge = stats_.gauge("overload.envoy.overload_actions.dummy_action.active",
+                                            Stats::Gauge::ImportMode::Accumulate);
+  Stats::Gauge& scaling_gauge = stats_.gauge("overload.envoy.overload_actions.dummy_action.scaling",
+                                             Stats::Gauge::ImportMode::Accumulate);
```

Comment:
Change this to use the scale_value gauge instead.

Reply:
Sorry, that was sloppy. Done, and the test is now passing.